docs/api/params.md

```diff
@@ -25,15 +25,22 @@ or by ID.
 Example without sequential:
 
 ```php
-$result= civicrm_api('UFMatch','Get', array('version' =>3, 'uf_id' => $user->uid);
-$contactid=$contact['values'][$result['id']]['contact_id'] );
+$result = civicrm_api('UFMatch', 'Get', array(
+  'version' => 3,
+  'uf_id' => $user->uid,
+));
+$contactid = $contact['values'][$result['id']]['contact_id'];
```

Comment:
There is a problem in the last line - `$contact` and `$result` are mixed up in the last line. (Pre-existing problem, of course.)